storage/src/vespa/storage/persistence/filestorage/filestormanager.cpp

```diff
@@ -259,7 +259,7 @@ FileStorManager::handlePersistenceMessage(
             msg->getType().getName().c_str(), disk);
 
         LOG_BUCKET_OPERATION_NO_LOCK(
-                getStorageMessageBucketId(*msg),
+                getStorageMessageBucketId(*msg).getBucketId(),
```

Comment:
Should be `getStorageMessageBucket(*cmd)`, though this code isn't compiled by default and therefore won't trigger a compilation error.